### PR TITLE
Adds missing newsletter styling to minimalist

### LINKED ISSUE #6713: Fullwidth newsletter form on desktop viewport
**Summary**
I notice the newsletter at document pages is fullwidth on both desktop and small viewports, it is inconsistent with white-space wrapped designs which has been adopted by most other components.

**Steps To Reproduce (STR)**

1.  Go to [web component doc page]( https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots)
2. Scroll down to the footer

**Actual behavior**

Screen Shot 2020-03-21 at 8 59 10 pm

**Expected behavior**

The newsletter form should be centred wrapped by 20px margin and adopted with responsive width. 

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by schalkneethling**
fix #6713 @mindy r?

### CODE REVIEW

File: kuma/static/styles/minimalist/main.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ Main stylesheet for the site, standalone
 @import '../components/_tags.scss';
 @import '../components/_errorlist.scss';
 @import '../components/_newsletter.scss';
+@import './components/_newsletter.scss';
```
Comment (peterbe) [inline kuma/static/styles/minimalist/main.scss]: Hehe. Perhaps it's time to name those two things differently :)

Comment (mindy) [reply kuma/static/styles/minimalist/main.scss]: Curious why the minimalist rules in `_newsletter.scss` aren't merged with the other `_newsletter.scss`?

Comment (schalkneethling) [reply kuma/static/styles/minimalist/main.scss]: Just have not had the time :(

Comment (mindy) [reply kuma/static/styles/minimalist/main.scss]: It's okay :)

**Pull request merged**